src/domain/orders/details/create-fulfillment/index.tsx

```diff
@@ -0,0 +1,314 @@
+import {
```

Comment:
@zakariaelas  Added the create fulfillment modal here. Should handle fulfillments for both orders, swaps and claims. Note, that in order for me to make this modal, I've merged in a couple of other PRs. Will add to the description, but just so you don't get shocked over all of the new files 😅

Reply:
Files to review are this one and `src/domain/orders/details/create-fulfillment/items-table.tsx`